### DISCUSSION TITLE
Where should I report bugs in syntax highlighting?

### DISCUSSION THREAD

**Discussion by simon-liebehenschel** [NvChad/NvChad#2021] [Q&A] [answered]
I've noticed that f-strings in Python are parsed correctly, by highlighting is wrong. The problem is that nested variables have the same color as a string itself, but the color must me different.

Where do I need to fill an issue?

- https://github.com/nvim-treesitter/nvim-treesitter/ ? 
- NvChad themes?
- pyright?

**Comment by siduck** [comment, marked as answer]
@AIGeneratedUsername  if it looks normal in non nvchad config then please make an issue in our base46 repo
